args.zig

```diff
@@ -309,7 +342,7 @@ pub fn ParseArgsResult(comptime Generic: type, comptime MaybeVerb: ?type) type {
         arena: std.heap.ArenaAllocator,
 
         /// The options with either default or set values.
-        options: Generic,
+        options: Generic = undefined,
```

Comment:
Please move this initialization to the instantiation of the structure and don't make it a implicit default. Especially as manual default initialization is done later, and this field is required to be non-`undefined` before use!

Reply:
Good point